src_c/display.c

```diff
@@ -2715,6 +2728,8 @@ static PyMethodDef _pg_display_methods[] = {
      METH_NOARGS, "provisional API, subject to change"},
     {"get_desktop_sizes", (PyCFunction)pg_get_desktop_screen_sizes,
      METH_NOARGS, "provisional API, subject to change"},
+    {"get_window_ID", (PyCFunction)pg_get_window_ID,
```

Comment:
Can we return a `Window` here?

Reply:
I might make this a private API.
I'll probably add the function to get the window to _sdl2.Window directly, and call it from_display() or something

Reply:
`Window.from_display()` sounds good to me. So `display` is not bound to `video`. display.c could export it in c_api so `video` can use it.